packages/opentelemetry-api/src/index.ts

```diff
@@ -30,6 +30,7 @@ export * from './metrics/MetricObservable';
 export * from './metrics/NoopMeter';
 export * from './metrics/NoopMeterProvider';
 export * from './metrics/ObserverResult';
+export * from './platform';
```

Comment:
hmm not sure how i feel about exporting our `_globalThis` variable... @mayurkale22 wdyt?

Reply:
I thought about using the window, but it won't be compatible with our global definition and I might simply endup of duplicating code.

Reply:
Nothing needs to be shared between API and core for this, so I think it's safe to have them defined separately. When users set options, they will most likely use `window.OTEL_SAMPLING_PROBABILITY` anyway, and accessing the global object through the API is not something we want to encourage.

Reply:
Only reason for `globalThis` is to allow code sharing between platforms. Since we already have code platform specific, I think it is safe to just use `window`.

Reply:
with "globalThis" we might have more advantages. It will be transparent for us to use it and don't worry if the window is there or not (react native, workers etc.). so if for any reason we will change the global in future it will simply match it. So instead of using window directly I thought it makes more sense to start using the api globalthis. It will work for now with window but might save us from problems in future.

Reply:
to be clear, my issue was not with the use of `globalThis`, but with exporting our own `global` from `api`. Anything exported from API will be very hard to remove once we go GA and exporting it from the API package implies that we think users should use it.